app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSigned.kt

```diff
@@ -35,7 +35,7 @@ class CheckOpeningHoursSigned (
           and access !~ private|no
           and (
             name or brand or noname = yes or name:signed = no
-            or amenity ~ recycling|toilets|bicycle_rental or leisure=park or barrier
+            or amenity ~ recycling|toilets|bicycle_rental|charging_station|social_facility or leisure=park or barrier
```

Comment:
Should ask for names of certain social facilities (food_bank|clothing_bank|soup_kitchen|dairy_kitchen) too and remove it from here